pandas/tseries/tests/test_resample.py

```diff
@@ -769,6 +769,36 @@ def test_resample_empty_dtypes(self):
                         # (ex: doing mean with dtype of np.object)
                         pass
 
+    def test_resample_loffset_arg_type(self):
+        # GH 13218, 15002
+        df = self.create_series().to_frame('value')
+        expected_means = [df.values[i:i + 2].mean()
+                          for i in range(0, len(df.values), 2)]
+        expected_index = self.create_index(df.index[0],
+                                           periods=len(df.index) / 2,
+                                           freq='2D')
+        # loffset coreces PeriodIndex to DateTimeIndex
```

Comment:
hmm that seems odd that a PI coerces when loffset is applied. is this happen in previous versions of pandas?